core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/RegistryResources.java

```diff
@@ -153,6 +153,13 @@
         public static final String SERVER_REGISTRY_KEY_PASSWORD = "Security.RegistryKeyStore.KeyPassword";
         public static final String SERVER_REGISTRY_KEYSTORE_TYPE = "Security.RegistryKeyStore.Type";
 
+        //Internal key store which is used for encryption and decryption purpose
```

Comment:
Can we modify this comment as follows?
// Internal key store properties which are used for encryption and decryption purposes.

Reply:
This is old code. All other places we have followed the same way. Therefore didn't change it inorder to have the consistency.